thepeg.sh

```diff
@@ -6,29 +6,33 @@ requires:
   - Rivet
   - GSL
   - fastjet
+  - pythia8
+  - HepMC
+  - lhapdf5
+  - cgal
+  - boost
 ---
 #!/bin/bash -e
 
-Pythia8='/cvmfs/alice.cern.ch/x86_64-2.6-gnu-4.1.2/Packages/pythia/v8186'
-HepMC='/cvmfs/alice.cern.ch/x86_64-2.6-gnu-4.1.2/Packages/HepMC/v2.06.09'
-LhaPDF='/cvmfs/alice.cern.ch/x86_64-2.6-gnu-4.1.2/Packages/lhapdf/v5.9.1'
-Cgal='/cvmfs/alice.cern.ch/x86_64-2.6-gnu-4.1.2/Packages/cgal/v4.4'
-Boost='/cvmfs/alice.cern.ch/x86_64-2.6-gnu-4.1.2/Packages/boost/v1_53_0'
+export LD_LIBRARY_PATH="${LHAPDF5_ROOT}/lib:${CGAL_ROOT}/lib:${BOOST_ROOT}/lib:${LD_LIBRARY_PATH}"
+export LIBRARY_PATH="${LHAPDF5_ROOT}/lib:${CGAL_ROOT}/lib:${BOOST_ROOT}/lib:${LIBRARY_PATH}"
+export LHAPATH="${LHAPDF5_ROOT}/share/lhapdf"
+export PERLLIB=/usr/share/perl5
 
-export LD_LIBRARY_PATH="${LhaPDF}/lib:${Cgal}/lib:${Boost}/lib:${LD_LIBRARY_PATH}"
-export LIBRARY_PATH="${LhaPDF}/lib:${Cgal}/lib:${Boost}/lib:${LIBRARY_PATH}"
-export LHAPATH="${LhaPDF}/share/lhapdf/PDFsets"
+rsync -a $SOURCEDIR/ ./
 
-cd "$SOURCEDIR"
+sed -i -e 's#@PYTHIA8_DIR@/xmldoc#@PYTHIA8_DIR@/share/Pythia8/xmldoc#' TheP8I/Config/interfaces.pl.in
+sed -i -e 's#@PYTHIA8_DIR@/xmldoc#@PYTHIA8_DIR@/share/Pythia8/xmldoc#' TheP8I/src/Makefile.am
+sed -i -e 's#@PYTHIA8_DIR@/xmldoc#@PYTHIA8_DIR@/share/Pythia8/xmldoc#' TheP8I/src/Makefile.in
 
-find . \
-  -name configure.ac -or \
-  -name aclocal.m4 -or \
-  -name configure -or \
-  -name Makefile.am -or \
-  -name Makefile.in \
-  -exec touch '{}' \;
-autoreconf -ivf
+#find . \
```

Comment:
Understood. Here, I left it in on purpose because Dario said he needed that part, while I did not now (on Ubuntu). And we might indeed need it for SLC 5 which we can't build and test yet because of the missing dependencies (as a result of the lack of python).